src/openzaak/components/autorisaties/admin_views.py

```diff
@@ -54,27 +55,40 @@ def get_initial_for_component(
     autorisaties: List[Autorisatie],
     spec: Optional[AutorisatieSpec] = None,
 ) -> List[Dict[str, Any]]:
-    if component not in [ComponentTypes.zrc, ComponentTypes.drc, ComponentTypes.brc]:
-        return []
+    _related_objs = {}
+    _related_objs_external = []
+
+    internal_autorisaties = []
+    for autorisatie in autorisaties:
+        try:
```

Comment:
this is also incorrect resolution if it works just on the UUID value :thinking: 

Probably better to use the loose-fk local-url mechanism here too.